### PR TITLE
Admin messages for prayers show the round's deity.

### PR DESCRIPTION
**Pull request by harmonyn**
Someone requested this in asay and it seemed simple enough. Will (definitely not) promote better RP for the chaplain. Cultist prayers obviously show Nar-Sie.

### CODE REVIEW

File: code/modules/admin/verbs/pray.dm [hunk 1]
```diff
@@ -20,10 +20,13 @@
...
 	var/font_color = "purple"
 	var/prayer_type = "PRAYER"
+	var/deity = "DEITY"
 	if(usr.job == "Chaplain")
 		cross = image('icons/obj/storage.dmi',"kingyellow")
 		font_color = "blue"
-		prayer_type = "CHAPLAIN PRAYER"
+		if (ticker && ticker.Bible_deity_name)
+			deity = ticker.Bible_deity_name
+		prayer_type = "CHAPLAIN PRAYER ([deity])"
```
Comment (lzimann) [inline code/modules/admin/verbs/pray.dm]: dunno if it's possible, but can you even have a chaplain pray without a deity? if so the message will be kinda weird with just ()...

Comment (bgobandit) [inline code/modules/admin/verbs/pray.dm]: maybe before they set their deity, in that case it'd say (DEITY), I guess I could change it to Space Jesus or something

Comment (lzimann) [inline code/modules/admin/verbs/pray.dm]: you could keep that prayer_type before the if, and if there's deity you just add " (deity)"

**Comment by TechnoAlchemisto** [comment]
It'd be nice for this to be for all prayers, since SM says "you hear the voice of (deity) in your head"

**Comment by Jalleo** [comment]
Only if pray is pray to "name of diety" (excluding all antags they pret to their leaders.

**Comment by yackemflam** [comment]
The clown needs honk mother

**Comment by MimicFaux** [comment]
And the mime his Silent Father

**Comment by Alek2ander** [comment]
Ratvar tho?

**Comment by bgobandit** [comment]
in all seriousness: I guess I can add Ratvar but the rest would be a ridiculous amount of checks for something relatively small

**Pull request merged**